src/client/game.ts

```diff
@@ -98,7 +101,7 @@ export default class ClientGame extends GameLoop {
     this.down.unsubscribe();
     this.up.unsubscribe();
   }
-
+  // this is not pretty
```

Comment:
```suggestion
  
```
Äre inte onödigt att ha såna här kommentarer? Gör issues om det är ett riktigt problem tänker jag

Reply:
eller till en `TODO: split this function into smaller sub-functions`